csv/src/main/java/org/apache/metamodel/csv/CsvDataContext.java

```diff
@@ -244,7 +227,7 @@ private static File createFileFromInputStream(InputStream inputStream, String en
         file = fileCandidate;
 
         final BufferedWriter writer = FileHelper.getBufferedWriter(file, encoding);
-        final BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream));
+        final BufferedReader reader = new BufferedReader(new InputStreamReader(inputStream, Charset.forName(encoding)));
```

Comment:
For the sake of consistency, would it be nicer to use `FileHelper.getBufferedReader(inputStream, encoding)`?